src/java/org/apache/cassandra/io/compress/ZstdCompressor.java

```diff
@@ -22,11 +22,14 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
+import com.github.luben.zstd.Zstd;
+
 /**
  * ZSTD Compressor
  */
 public class ZstdCompressor extends ZstdCompressorBase implements ICompressor
 {
+    public static final int BEST_COMPRESSION_LEVEL = Zstd.maxCompressionLevel();
```

Comment:
Thank you for making `ZstdCompressor` backward compatible! 

Maybe add a comment to explain why there are 2 different `BEST_COMPRESSION_LEVEL`